core/src/main/scala/org/apache/spark/storage/BlockManagerMessages.scala

```diff
@@ -36,6 +36,9 @@ private[spark] object BlockManagerMessages {
   case class ReplicateBlock(blockId: BlockId, replicas: Seq[BlockManagerId], maxReplicas: Int)
     extends ToBlockManagerSlave
 
+  case object DecommissionBlockManager
+    extends ToBlockManagerSlave
```

Comment:
Let's make this one-liner since it's simple.

Reply:
Updated.